deploy/helm/elastic-agent/templates/agent/k8s/_pod_template.yaml

```
@@ -149,6 +149,10 @@ template:
           {{- with ($presetVal).outputs }}
           {{- range $outputName, $outputVal := . -}}
           {{- (include (printf "elasticagent.output.%s.preset.envvars" ($outputVal).type) (list $ $outputName $outputVal)) | nindent 10 }}
+          {{- $outputSSLConfig := dig "ssl" dict $outputVal -}}
+          {{- if not (empty $outputSSLConfig) }}
+          {{- (include "elasticagent.output.render.sslconfig.envvars" (list $outputSSLConfig $outputName $agentName) | nindent 10) }}
+          {{- end }}
```

if already checks for empty values so this can be simplified like
```suggestion
          {{- $outputSSLConfig := dig "ssl" dict $outputVal -}}
          {{- if $outputSSLConfig }}
          {{- (include "elasticagent.output.render.sslconfig.envvars" (list $outputSSLConfig $outputName $agentName) | nindent 10) }}
          {{- end }}
```